1_fetch/src/get_daily_nwis_data.R

```diff
@@ -0,0 +1,63 @@
+get_daily_nwis_data <- function(site_info,parameter,stat_cd_select) {
```

Comment:
eventually, we'll probably want the start and end date as arguments here. Comment may apply to other functions, too

Reply:
Thanks, I agree! For example, specifying an end date would be useful for ensuring consistent builds across users. I've added `start_date` and `end_date` as arguments to `get_daily_nwis_data` and `get_inst_nwis_data`. What I'm thinking is that I'll keep "" as defaults for those arguments (this is what dataRetrieval does), but within our `1_fetch.R` phase file, specifying `dummy_date` as the `end_date` within the target builds for the daily and instantaneous data. That way `dummy_date` is useful for us as an indicator of last build/data download. How does that sound?

Reply:
I think that's a great idea